docs/how-tos/installation_notes_nxvalidate.md

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
Did you want to keep this? So far it is only avaialble in the docs through the relative link in validate_nexus-file.md

Reply:
ah - should this otherwise be placed in references? Is this the intention of the folder? Just wanted to keep it as note, but not as "how-to".

Reply:
The reason we have these categories is because we are using the [Diátaxis framework](https://diataxis.fr/). I gues you can leave it where it is, it's also not really a reference or an explanation

Reply:
Maybe explanation (i.e., learn) would be the best place actually..

Reply:
ok, ill keep it.